supabase/migrations/20260418000000_add_performance_indexes.sql

```
@@ -0,0 +1,13 @@
+-- Migration to add missing indexes on foreign keys and columns used in WHERE/JOIN clauses
+-- Created for performance optimization
+
+CREATE INDEX IF NOT EXISTS idx_expenses_category_id ON expenses(category_id);
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)